tests/ProxyManagerTest/Functional/AccessInterceptorScopeLocalizerFunctionalTest.php

```diff
@@ -523,7 +520,7 @@ public function testWillInterceptAndReturnEarlyOnVoidMethod() : void
                         VoidCounter $instance,
                         string $method,
                         array $params,
-                        ?bool & $returnEarly
+                        ?bool &$returnEarly
```

Comment:
These should be reverted: `& $returnEarly` is the correct formatting when using by-ref params (which are otherwise much more elusive to the eye)

Reply:
Note: the CS rule changing this can be disabled, or maybe we can upgrade to a newer `doctrine/coding-standard` version to see if that already fixes it.